src/components/PageInfo/PageInfo.tsx

```diff
@@ -0,0 +1,12 @@
+import { useParams } from 'react-router-dom';
+import { tabs } from '../../api/tabs';
```

Comment:
suggestion: you'd rather pass them as props, it looks more natural

Reply:
okey